xcodebuild/build.go

```diff
@@ -154,26 +168,38 @@ func (c *CommandBuilder) cmdSlice() []string {
 	if c.scheme != "" {
 		slice = append(slice, "-scheme", c.scheme)
 	}
+
 	if c.configuration != "" {
 		slice = append(slice, "-configuration", c.configuration)
 	}
 
 	if c.forceDevelopmentTeam != "" {
 		slice = append(slice, fmt.Sprintf("DEVELOPMENT_TEAM=%s", c.forceDevelopmentTeam))
 	}
+
 	if c.forceProvisioningProfileSpecifier != "" {
 		slice = append(slice, fmt.Sprintf("PROVISIONING_PROFILE_SPECIFIER=%s", c.forceProvisioningProfileSpecifier))
+	} else if c.disableProvisioningProfileSpecifier {
+		slice = append(slice, fmt.Sprintf("PROVISIONING_PROFILE_SPECIFIER="))
 	}
+
 	if c.forceProvisioningProfile != "" {
 		slice = append(slice, fmt.Sprintf("PROVISIONING_PROFILE=%s", c.forceProvisioningProfile))
 	}
+
 	if c.forceCodeSignIdentity != "" {
 		slice = append(slice, fmt.Sprintf("CODE_SIGN_IDENTITY=%s", c.forceCodeSignIdentity))
 	} else if c.disableCodesign {
```

Comment:
in case of `disableCodesign ` i think we should also: `PROVISIONING_PROFILE_SPECIFIER="" & PROVISIONING_PROFILE =""`